### PR TITLE
Style update wizard radiobuttons and inputfields

### PR DESCRIPTION
**Pull request by rolf-yoast**
## Summary

This PR can be summarized in the following changelog entry:

* [@yoast/components] Updates the configuration wizard to match the new design

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch and the [corresponding PR on the plugin](https://github.com/Yoast/wordpress-seo/pull/15388).
* Open the configuration wizard
* Check the different steps to see if they match the new design

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  * Configuration wizard

## UI changes
* [x] This PR changes the UI in the plugin. I have added the 'UI change' label to this PR.

## Quality assurance

* [x] I have tested this code to the best of my abilities
* [ ] I have added unittests to verify the code works as intended

Fixes https://yoast.atlassian.net/browse/FRO-24

### CODE REVIEW

File: apps/components/webpack.config.js [hunk 1]
```diff
@@ -11,7 +11,8 @@ module.exports = {
 		"babel-polyfill",
 
 		// Compile the @yoast/configuration-wizard scss
-		"@yoast/configuration-wizard/src/configuration-wizard.scss",
+		// Needs to be fixed
+		// "@yoast/components/src/configuration-wizard/configuration-wizard.css",
```
Comment (rolf-yoast) [inline apps/components/webpack.config.js:15]: Needs to be fixed in this PR or can this be removed?

**Comment by rolf-yoast** [comment]
CR & acceptance 🚧
See the comment

File: apps/components/webpack.config.js (hunk 1 above)
Comment (JoseeWouters) [reply apps/components/webpack.config.js:15]: There's a new issue for that: https://yoast.atlassian.net/browse/FRO-106

**Comment by rolf-yoast** [comment]
CR & Acceptance 👍

**Comment by igorschoester** [comment]
This PR is reverted in https://github.com/Yoast/javascript/pull/770
The milestone and changelogs are not correct here. This changelog is not correct. I'm just changing the milestone to 14.6, but it is not actually included.

There will be another revert revert PR later on a feature branch that will include this code.

**Pull request merged**